tsconfig.json

```diff
@@ -1,43 +1,20 @@
 {
+  "version": "1.6.2",
   "compilerOptions": {
-    "target": "ES5",
-    "experimentalDecorators": true,
+    "target": "es5",
+    "module": "commonjs",
+    "declaration": false,
+    "noImplicitAny": false,
+    "removeComments": true,
+    "noLib": false,
     "emitDecoratorMetadata": true,
-    "module": "umd",
-    "outDir": "./app/js/"
+    "experimentalDecorators": true,
+    "sourceMap": true,
+    "listFiles": false,
+    "outDir": "dist"
```

Comment:
seems like the outDir option is not really used, should we remove it? Or rename it to build to reflect where the bundle files actually go?